portal/apps/org.wso2.carbon.iam.portal.common/src/main/fragments/all-profiles/all-profiles.hbs

```diff
@@ -0,0 +1,63 @@
+{{fragment "org.wso2.carbon.uuf.common.foundation.ui.jquery-validation"}}
+{{js "js/all-profiles.js"}}
+{{#if success}}
+    {{#if message}}
+        <div class="alert alert-success" role="alert">
+            <i class="icon fw fw-success"></i><strong>{{i18n "profile.success"}}</strong>
+            {{i18n message}}
+            <button type="button" class="close" aria-label="close" data-dismiss="alert">
+                <span aria-hidden="true">
+                    <i class="fw fw-cancel"></i>
+                </span>
+            </button>
+        </div>
+    {{/if}}
+{{else}}
+    {{#if message}}
+        <div class="alert alert-danger" role="alert">
+            <i class="icon fw fw-error"></i><strong>{{i18n "profile.failure"}}</strong>
+            {{i18n message}}
+            <button type="button" class="close" aria-label="close" data-dismiss="alert">
+                <span aria-hidden="true">
+                    <i class="fw fw-cancel"></i>
+                </span>
+            </button>
+        </div>
+    {{/if}}
+{{/if}}
+
+<div>
+
+    <br class="settings-content">
```

Comment:
Usually br tag used in presentation/layout is too rigid. Can not apply much of the CSS and change things.
(this comment is just for information only, as this code anyway will be thrown away in favor of new UI framework)